### PR TITLE
Add dependabot guide

### LINKED ISSUE #129: Add guide for setting up dependabot
Dependabot is a highly useful tool for managing `package-lock.json` and `composer.lock` file changes, highlighting when updates are available and automating those updates given developer approval. It also has the advantage of handling conflicts nicely by rebasing it's PRs automatically when the target branch changes, removing a potential headache for devs.

https://dependabot.com/

### PR DESCRIPTION
**Pull request by roborourke**
Fixes #129

### CODE REVIEW

File: other-docs/guides/automatic-updates.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Setting Up Dependabot
+
+To get started create a `dependabot.yml` file in your project root.
+
+The minimum recommended configuration for Altis is as follows:
+
+```yaml
+version: 2
+updates:
+  # Enable version updates for npm
```
Comment (rmccue) [inline other-docs/guides/automatic-updates.md:16]:
```suggestion
  # Enable version updates for Composer
```

File: other-docs/guides/automatic-updates.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - package-ecosystem: "composer"
+    # Look for `composer.json` and `composer.lock` files in the `root` directory
+    directory: "/"
```
Comment (rmccue) [inline other-docs/guides/automatic-updates.md:19]: Isn't this a default?

File: other-docs/guides/automatic-updates.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    # Check packagist registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```
Comment (rmccue) [inline other-docs/guides/automatic-updates.md:22]: Can this be live instead? AFAIK that's supported for Composer.

File: other-docs/guides/automatic-updates.md (hunk 2 above)
Comment (roborourke) [reply other-docs/guides/automatic-updates.md:19]: Nope, required field according to their docs

File: other-docs/guides/automatic-updates.md (hunk 3 above)
Comment (roborourke) [reply other-docs/guides/automatic-updates.md]: So it's not in GitHub's documentation anywhere, feels a little risky but if we have some examples already working with this then 👍 

Updated.

File: other-docs/guides/automatic-updates.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+The minimum recommended configuration for Altis is as follows:
+
+```yaml
+version: 2
+updates:
+  # Enable version updates for Composer
+  - package-ecosystem: "composer"
+    # Look for `composer.json` and `composer.lock` files in the `root` directory
+    directory: "/"
+    # Create pull requests as soon as updates are made avilable
```
Comment (rmccue) [inline other-docs/guides/automatic-updates.md:20]:
```suggestion
    # Create pull requests as soon as updates are made available
```

File: other-docs/guides/automatic-updates.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Automating Updates
+
+The recommended approach to automating updates to Altis is to use GitHub's Dependabot feature.
```
Comment (rmccue) [inline other-docs/guides/automatic-updates.md:3]:
```suggestion
The recommended approach to automating updates to Altis is to use GitHub's Dependabot feature. This feature is provided for free by GitHub.
```

**Pull request merged**